addon/package.json

```diff
@@ -45,7 +45,7 @@
     "@types/ember-test-helpers": "^1.0.10",
     "@types/ember-testing-helpers": "^0.0.4",
     "@types/ember__test-helpers": "^2.6.1",
-    "@types/qunit": "^2.11.2",
+    "@types/qunit": "^2.19.9",
```

Comment:
this changed because between the different test apps we have different qunit types versions, and it's _just easier_ to have these deps match, than to fix subtrees of the dep graph